qutip/tests/test_gates.py

```diff
@@ -70,6 +73,71 @@ def testSwapGate(self):
         psi_res = swap() * swap() * psi_in
         assert_((psi_in - psi_res).norm() < 1e-12)
 
+    def test1QubitGates(self):
+         """
+         gates: X,Y,Z,S,T gates
+         """
+         N = 7
+         for g in [x_gate, y_gate, z_gate, s_gate, t_gate]:
+
+             a, b = np.random.rand(), np.random.rand()
+             psi1 = (a * basis(2, 0) + b * basis(2, 1)).unit()
+             psi2 = g() * psi1
+
+             psi_rand_list = [rand_ket(2) for k in range(N)]
+
+             for m in range(N):
+
+                 psi_in = tensor([psi1 if k == m else psi_rand_list[k]
+                                  for k in range(N)])
+                 psi_out = tensor([psi2 if k == m else psi_rand_list[k]
+                                   for k in range(N)])
+
+                 G = g(N, m)
+                 psi_res = G * psi_in
+
+                 assert_((psi_out - psi_res).norm() < 1e-12)
+
+    def testCtrld1QubitGates(self):
```

Comment:
I'm strongly against arbitrary contraction in names, and that counts too in test descriptions - I'd have called this test `test_controlled_one_qubit_gates` (though I know `qutip` is a bit of a fiend from randomly chopping bits out of words in keywords all over the place).

Given that you've been chopping and changing everything in the name of pep8, I'm surprised you went along with the rest of the file's camel-cased functions!

Reply:
Fixed on ad74aeb